<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleChat</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/background/res/background.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loginButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>156</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/background/res/loginbut.png);
color: rgb(255, 255, 255);
border:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();

font: 9pt &quot;微软雅黑&quot;;
border:0px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>注册新用户</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="accountEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>182</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
border-top-left-radius:0px;
border-top-right-radius:0px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findpwdButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
background-image: url();
border:0px;</string>
    </property>
    <property name="text">
     <string>找回账号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>463</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background:transparent;
border:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="miniButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background:transparent;
border:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
